.gitlab/ci.sh

```diff
@@ -61,18 +63,18 @@ args=(
 run cabal v2-build ${args[@]} cabal-install
 
 mkdir "$CI_PROJECT_DIR/out"
-cp "$(cabal list-bin ${args[@]} cabal-install:exe:cabal)" "$CI_PROJECT_DIR/out/cabal"
+cp "$(cabal list-bin ${args[@]} cabal-install:exe:cabal)" "$CI_PROJECT_DIR/out/cabal$EXE_EXT"
 cp dist-newstyle/cache/plan.json "$CI_PROJECT_DIR/out/plan.json"
 cd "$CI_PROJECT_DIR/out/"
 
 # create tarball/zip
 TARBALL_PREFIX="cabal-install-$("$CI_PROJECT_DIR/out/cabal" --numeric-version)"
 case "${TARBALL_EXT}" in
     zip)
-        zip "${TARBALL_PREFIX}-${TARBALL_ARCHIVE_SUFFIX}.${TARBALL_EXT}" cabal plan.json
+        zip "${TARBALL_PREFIX}-${TARBALL_ARCHIVE_SUFFIX}.${TARBALL_EXT}" "cabal$EXE_EXT" plan.json
```

Comment:
(I'd prefer braced variable references (`${EXE_EXT}`), if just for consistency.)

Reply:
yeah, agree

Reply:
```suggestion
        zip "${TARBALL_PREFIX}-${TARBALL_ARCHIVE_SUFFIX}.${TARBALL_EXT}" "cabal${EXE_EXT}" plan.json
```